src/promptflow/promptflow/_sdk/_serving/flow_invoker.py

```diff
@@ -36,8 +37,7 @@ class FlowInvoker:
     def __init__(
         self, flow: str, connection_provider: [str, Callable] = None, streaming: Union[Callable[[], bool], bool] = False
     ):
-        self.flow_dir = flow
-        self.flow_entity = load_flow(self.flow_dir)
+        self.flow_entity = flow if isinstance(flow, Flow) else load_flow(source=flow)
```

Comment:
`flow` parameter annotation is only `str` in the interface.

Reply:
Yes, I'm not sure if we need to expose it.

Reply:
added